python/pyspark/sql/functions.py

```diff
@@ -4524,10 +4524,7 @@ def approx_percentile(
     of `col` values is less than the value or equal to that value.
 
 
-    .. versionadded:: 3.1.0
-
-    .. versionchanged:: 3.4.0
-        Supports Spark Connect.
```

Comment:
Let's probably then add a `Notes` section and say that it's supported in Spark Connect.

Reply:
done

Reply:
I am fine with this notes, we don't have it for other newly added functions tho.